Makefile

```diff
@@ -69,24 +69,33 @@ zip-prod: vendor dist .config.prod.yml
 	@$(call zip_it,.config.prod.yml,${PACKAGE}.zip)
 
 # target: build                                  - Setup PHP & Node.js locally
-build: vendor
+build: vendor tools-vendor
 
 composer.phar:
 	@php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');";
+	@php -r "copy('https://getcomposer.org/installer', 'tools/composer-setup.php');";
```

Comment:
IMO not required, as composer-setup.php can be called once to install composer.phar